src/components/tutorial/CompletionConfetti.astro

```diff
@@ -0,0 +1,24 @@
+<script>
+	if (
+		window.location.pathname.endsWith('/tutorial/6-islands/3/') &&
```

Comment:
Can the pathname check be left out since it's added as a component directly to the page?

Reply:
Ah, I should probably have added a comment explaining this slightly eccentric code — will do that now.

We glob these pages so this script will end up everywhere. Added the check to target a specific page and avoid downloading the confetti lib unnecessarily (hence the dynamic import).